src/bionics_ui.cpp

```diff
@@ -121,6 +124,48 @@ std::string build_bionic_powerdesc_string( bionic const &bio )
     return power_desc.str();
 }
 
+void show_bionics_tabs( WINDOW *win, size_t active_num, size_t passive_num,
```

Comment:
The verb `draw` seems to be used in the code for purposes of drawing much more often, than `show`.